Description: Disable tests fiddling around with files in debian/*. Not wanted by Debian maintainer.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
Forwarded: https://gitlab.com/ubports/development/core/qtmir/-/issues/2

Index: qtmir-0.7.0/tests/CMakeLists.txt
===================================================================
--- qtmir-0.7.0.orig/tests/CMakeLists.txt
+++ qtmir-0.7.0/tests/CMakeLists.txt
@@ -10,8 +10,8 @@ add_subdirectory(framework)
 add_subdirectory(mirserver)
 add_subdirectory(modules)
 
-add_test(
-    NAME requires-provides
-    COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/check-requires-provides.sh
-    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-)
+#add_test(
+#    NAME requires-provides
+#    COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/check-requires-provides.sh
+#    WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+#)
